.github/workflows/ash.yml

```diff
@@ -0,0 +1,30 @@
+name: ASH
+
+on:
+  workflow_dispatch:
```

Comment:
Do we want to run this on a schedule in addition to on demand?

Reply:
Not yet, we want to be in a position where we have limited the number of raised issues first